src/golang/lib/models/shared/service.go

```diff
@@ -37,7 +36,12 @@ const (
 	// Cloud integrations
 	AWS Service = "AWS"
 
-	DemoDbIntegrationName = "aqueduct_demo"
+	// Built-in integrations
+	Aqueduct     Service = "Aqueduct"
```

Comment:
This refers to our default compute engine service right?

Reply:
yes